EngineTests/Core/StringUtils.cpp

```diff
@@ -32,4 +32,31 @@ TEST(StringUtils, BytesString)
 	TestSize = 1310720; // 2^20 + (2^20)/4
 	TestSizeString = StringUtils::CreateBytesString(TestSize);
 	EXPECT_STREQ(TestSizeString.c_str(), "1.25 MB"); // 1.13 = 1 + 1/4 (rounded to two decimal)
+}
+
+TEST(StringUtils, PathConcat)
```

Comment:
Are we keeping this test in our source code?

Reply:
Yes of course, it's part of the `EngineTests` which is a set of unit tests which we run during CI.

Reply:
Oh sorry ya that makes a lot of sens :P